java/sage/MediaServerRemuxer.java

```diff
@@ -33,8 +33,10 @@
 public class MediaServerRemuxer
 {
   private static final int TS_ALIGN = 188;
-  private static final int MAX_TRANSFER = 33088;
-  private static final int MAX_INIT_BUFFER_SIZE = 20971400;
+  // The transfers need to be 188 byte aligned.
+  private static final int MAX_TRANSFER = 176 * TS_ALIGN;
```

Comment:
I do like how you show which values are multiple of 188 now; but there's still seemingly random values in here such as 176, 111550, 8388608 (which is 8MB, so just add a comment after it saying 8MB for clarity), 27888.  I'd be concerned changing them at a later time because I wouldn't know why those values were picked.  I'm not going to press you on it; so you can leave it this way if you want, but it's not a bad idea to be more clear about constant values like that if there's a reason behind them (and I quite often just put stuff like "random large buffer size", when I'm just picking some big number out of the air; then someone later would know there's no concern with changing it).